docs/clients/arming_handler.rst

```diff
@@ -0,0 +1,149 @@
+        ArmingHandlerClient armingHandler(0);
+
+        // Use the Arming Handler Client
+        armingHandler.always_armed_.get(com)
+
+        // Insert code for interfacing with hardware here  
+    }
+
+Matlab
+~~~~~~
+
+To use the Arming Handler client in Matlab, all Vertiq communication code must be included in your
+path. This allows the creation of a ArmingHandlerClient object. See the Message Table below for available messages.
+All message strings use the Short Names. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the ArmingHandlerClient is:
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+    % Make a ArmingHandlerClient object with obj_id 0
+    armingHandler = ArmingHandlerClient(’com’,com);
+    % Use the ArmingHandlerClient object
+    alwaysArmed = armingHandler.get(’always_armed’);
+
+Python
+~~~~~~
+
+To use the Arming Handler Client in Python, include ``iqmotion`` and create a module that has the Arming Handler Client within it's firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the Arming Handler Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0)
+    
+    always_armed_status = |variable_name|.get("arming_handler", "always_armed") 
+    print(f"Always armed status: {always_armed_status}")
+
+Message Table
+~~~~~~~~~~~~~
+
+Type ID 86 | Arming Handler
+
++--------+-------------------------------------+-----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Sub ID | Short Name                          | Access          | Data Type | Unit      | Note                                                                                                                                                                                                                                   |
++========+=====================================+=================+===========+===========+========================================================================================================================================================================================================================================+
+| 1      | always_armed                        | get, set, save  | uint8     |           | If True, handler will be automatically armed.                                                                                                                                                                                          |
```

Comment:
I may change the description to "If True, handler will automatically arm when the first throttle message is received."